python/fio/mocks/poisson-rate-submission_input.yaml

```diff
@@ -0,0 +1,11 @@
+name: poisson-rate-submit
```

Comment:
yeah, I was pretty sure these weren't mocks, but I couldn't recall the correct name. Should I change the name to `fixtures` and put it in a parent directory called `tests`?

Reply:
IMO neither "mocks" or "fixtures" means anything to me, and as a newcomer I won't quickly understand the intention or use of the files in directories named like this. Now that may be because I don't come from a software engineering background, so take it with a grain of salt. But my gut is to use an obvious naming convention like "tests" instead. Not important to block a merge, but worth discussion as part of #20